AUTHORS

```diff
@@ -4,3 +4,4 @@
 #   Name/Organization <email address>
 
 Google Inc.
+Sebastian Kreft
```

Comment:
the other PR landed so might have some conflicts to resolve on this one

Reply:
Thanks for the heads up. I updated my fork with the changes.